### DISCUSSION TITLE
Where can i get the mouse and other species's expression data?

### DISCUSSION THREAD

**Discussion by DOSUP** [Open-Systems-Pharmacology/Forum#1125] [Questions & Problems] [answered]
Thank you for viewing this.
I have been conducting tutorials (modeling biologics ADC) and stuck in some problems.
One of them, the tutorial show me that add TENB2 as a protein binding partner from "database query". However, I couldn't find expression data regarding mouse(only found human data) in Github. Although Modeling file 1 includes mouse expression data, I want to know how can I import expression data files of other species. please inform me the way.

**Comment by HenrikCordes** [comment, marked as answer]
Dear @DOSUP ,

how to link an expression database to PK-Sim is described here:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-options#template-database-path

Currently only a gene expression database for human is available. 
We are preparing a release for gene expression databases for various pre-clinical species #1096 

As soon as this is available, we will announce it in the forum.

Kind regards,
Henrik
